src/OrchardCore/OrchardCore.DisplayManagement.Liquid/Tags/FluidTagHelper.cs

```diff
@@ -88,12 +88,14 @@ public static async ValueTask<Completion> WriteToAsync(string identifier, List<F
 
             if (content != null)
             {
+                var output = content.ToString();
                 tagHelperOutput = new TagHelperOutput(
                     identifier,
-                    outputAttributes, (_, e) => Task.FromResult(new DefaultTagHelperContent().AppendHtml(content))
+                    outputAttributes, (_, e) => Task.FromResult(new DefaultTagHelperContent().AppendHtml(output))
```

Comment:
I think the issue here is that the same buffer is used twice, I wonder why? @jtkech

Reply:
Maybe I made a mistake when I refactored it too